src/main/java/org/jenkinsci/plugins/runselector/context/RunSelectorContext.java

```diff
@@ -143,6 +148,58 @@ public boolean isVerbose() {
     }
 
     /**
+     * @param projectName the project name to copy from.
+     */
+    public void setProjectName(String projectName) {
+        this.projectName = projectName;
+    }
+
+    /**
+     * The project name to copy from.
+     * Be aware that this might be different from the full name of the project
+     * as it might be specified with relative expression.
+     * 
+     * @return the project name to copy from.
+     */
+    @Nonnull
```

Comment:
I'm not convinced that it is never null. No guarantee setProjectName() gets invoked in API users, null can be also passed there